src/build.py

```diff
@@ -693,6 +693,12 @@ def AllSources():
     ]
 
 
+def ClobberOneDir(work_dir):
```

Comment:
I'm not crazy about the name of this function.

How about `RemoveBuildDir` .. or `RemoveIfIsBot`?   Not great alternatives.

Reply:
Yeah the "clobber" name goes way back to the original buildbot-based system, so "clobber" kind of implies that it has meaning only on bots. `RemoveIfBot` seems fine given how simple it is.